### ISSUE TITLE
Can it be used for k33

### ISSUE THREAD

**Issue by twosson** [madMAx43v3r/chia-plotter#478]
Can it be used for k33

**Comment by VertiHydro** [comment]
No. Not yet. _Please_ check existing issues for duplicates before making a new one.
See existing issues. #25, #65, #217, #416.
